src/components/LayoutTree.tsx

```diff
@@ -890,6 +1041,26 @@ const LayoutTree = () => {
     treeThoughts,
   ])
 
+  // Calculate isGreaterThought at LayoutTree level
+  const cursorIsGreaterThanPrevSibling = useSelector(state => {
```

Comment:
Doesn't this just need to compare the thought with its parent? No need to use `treeThoughtsPositioned` to access the render order. 

If so, this can go in `ThoughtNode`.